hera_opm/mf_tools.py

```diff
@@ -603,38 +603,11 @@ def build_lstbin_makeflow_from_config(config_file, mf_name=None, work_dir=None):
         base, ext = os.path.splitext(cf)
         fn = "{0}.mf".format(base)
 
-    # get the work directory
-    if work_dir is None:
-        work_dir = os.getcwd()
-    else:
-        work_dir = os.path.abspath(work_dir)
-    makeflowfile = os.path.join(work_dir, fn)
-
     # determine whether or not to parallelize
     parallelize = str(get_config_entry(config, "LSTBIN_OPTS", "parallelize", required=True)) == 'True'
     parent_dir = str(get_config_entry(config, "LSTBIN_OPTS", "parent_dir", required=True))
-
-    if parallelize:
-        # get LST-specific config options
-        dlst = get_config_entry(config, 'LSTBIN_OPTS', 'dlst', required=True)
-        if dlst == "None":
-            dlst = None
-        else:
-            dlst = float(dlst)
-        lst_start = float(get_config_entry(config, 'LSTBIN_OPTS', 'lst_start', required=True))
-        ntimes_per_file = int(get_config_entry(config, 'LSTBIN_OPTS', 'ntimes_per_file', required=True))
-
-        # pre-process files to determine the number of output files
-        datafiles = get_config_entry(config, "LSTBIN_OPTS", "data_files", required=True)
-        datafiles = [df.strip("\\\'") for df in datafiles]
-        datafiles = map(lambda df: str(df), datafiles)
-        datafiles = map(lambda df: sorted(glob.glob(os.path.join(parent_dir, df))), datafiles)
-
-        output = lstbin.config_lst_bin_files(datafiles, dlst=dlst, lst_start=lst_start,
-                                             ntimes_per_file=ntimes_per_file)
-        nfiles = len(output[3])
-    else:
-        nfiles = 1
+    work_dir = parent_dir
```

Comment:
Do we want to check that `parent_dir` is in `kwargs`? (I'm genuinely asking, since I don't usually write functions with `kwargs`.)

Reply:
`parent_dir` isn't fed through `kwargs`, its fed via the `config` file

Reply:
Oh right, good point.